File: module_tablayout/src/main/java/com/example/module/tablayout/multiple_list/bean/Person.java
package com.example.module.tablayout.multiple_list.bean;

import com.chad.library.adapter.base.entity.MultiItemEntity;
import com.example.module.tablayout.multiple_list.ExpandableItemAdapter;

/**
 * Created by luoxw on 2016/8/10.
 */

public class Person implements MultiItemEntity {
    public Person(String name, int age) {
        this.age = age;
        this.name = name;
    }

    public String name;
    public int age;

    @Override
    public int getItemType() {
        return ExpandableItemAdapter.TYPE_PERSON;
    }
}

File: module_kotlin/src/main/java/com/module/kt/bean/Person.kt
package com.module.kt.bean

import android.util.Log

/**
 *  Created by wangwei on 2020/4/28.
 */

class Person {
    var name: String? = null
    var age: Int? = null
    fun speak() {
        Log.w("TAG", "武汉加油,中国加油")
    }
}